core-bundle/tests/Picker/TablePickerProviderTest.php

```diff
@@ -468,6 +468,8 @@ private function createMenuTableProvider(array $modules, string $current, ItemIn
 
         if ($menu) {
             $expectedItems[] = ['picker'];
+        } else {
+            $menu = $this->createMock(ItemInterface::class);
         }
```

Comment:
Haha, did the absolute same here: https://github.com/contao/contao/pull/1924/files#diff-9864f398aca7e002108ecdfe8849b77a